test/tools/fixtures/process_resource_script.in.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+const driverPath = DRIVER_SOURCE_PATH;
+const func = FUNCTION_STRING;
+const name = NAME_STRING;
```

Comment:
Could we choose a more descriptive name for `name`? `name` of what?

Reply:
(same for `func`, although `func` is a bit more clear).

Reply:
I've changed it to `scriptName`!